lib/sidekiq/grouping/supervisor.rb

```diff
@@ -4,10 +4,17 @@ module Supervisor
       class << self
         include Sidekiq::Grouping::Logging
 
+        if Celluloid::VERSION >= '0.17'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.